kaali.js

```diff
@@ -0,0 +1,1371 @@
+// Copy Script Link
+content = document.querySelector("#script-link");
+if (content != null) {
+  content.addEventListener("click", () => {
+    navigator.clipboard.writeText(
+      '<script src="https://kaaliui.netlify.app/kaali.js"></>'
+    );
+
+    var tooltip = document.querySelector("#scripttooltip");
+
+    tooltip.innerHTML = "Copied";
+  });
+  content.addEventListener("mouseout", () => {
+    var tooltip = document.querySelector("#scripttooltip");
+    tooltip.innerHTML = "Copy to Clipboard";
+  });
+}
+// ---------------------------------------------------------------------------------------------
+
+// Copy css link
+content = document.querySelector("#css-link");
+if (content != null) {
+  content.addEventListener("click", () => {
+    navigator.clipboard.writeText(
+      '<link rel="stylesheet" href="https://kaaliui.netlify.app/kaali.css">'
+    );
+    let tooltip = document.querySelector("#mytooltip");
+
+    tooltip.innerHTML = "Copied";
+  });
+
+  content.addEventListener("mouseout", () => {
+    let tooltip = document.querySelector("#mytooltip");
+    tooltip.innerHTML = "Copy to Clipboard";
+  });
+}
+// ---------------------------------------------------------------------------------------------
+
+// Making navbar responsive STEP 1
+if (
+  document.querySelector(".fas.fa-bars") != null &&
+  document.querySelector(".fas.fa-bars").style.display != "none"
+) {
+  document.querySelector(".fa-bars").addEventListener("click", () => {
+    document.querySelector(".navbar-ul").style.display = "flex";
+    document.querySelector(".fas.fa-bars").style.display = "none";
+    document.querySelector(".close-menu").style.display = "block";
+  });
+}
+
+if (
+  document.querySelector(".close-menu") != null &&
+  document.querySelector(".close-menu").style.display != "none"
+) {
+  document.querySelector(".close-menu").addEventListener("click", () => {
+    document.querySelector(".navbar-ul").style.display = "none";
+    document.querySelector(".fas.fa-bars").style.display = "block";
+    document.querySelector(".close-menu").style.display = "none";
+  });
+}
+// ---------------------------------------------------------------------------------------------
+
+// load alert.html for desktop
+$(window).load(function () {
```

Comment:
Should convert all the Jquery code to plain JS.

Reply:
You should checkout Vikas JS file: https://github.com/vikasvk11/css-comp-lib-review/blob/development/index.js and can see how to create the copy to clipboard function,